VideoWeb/VideoWeb/ClientApp/src/app/shared/header/header.component.html

```diff
@@ -4,7 +4,11 @@
 
     <div class="moj-header__logo">
 
-    <app-header-logo-svg></app-header-logo-svg>
+    <img role="presentation" id="header-logo-scot" class="moj-header__logotype-crest" *ngIf="hearingVenueIsScottish$ | async;else ukCrestSvg" src="/assets/images/scot-crest-white.png" height="40" [attr.alt]="'participants-panel.scottish-crest' | translate"/>
```

Comment:
Could you move the img inside another template and put the ngIf inside a ng-container like this
```suggestion
    <ng-container
  *ngIf="hearingVenueIsScottish$ | async; then scottishCrest; else ukCrest">
</ng-container>
```

I feel otherwise the logic gets a bit lost amongst the other attributes.